get-report.py

```diff
@@ -6,11 +6,11 @@
 
 sf = SalesforceConnection()
 
-path = "/services/data/v46.0/analytics/reports/00O6f000007udDgEAI"
+path = "/services/data/v46.0/analytics/reports/00O6f000007v2YZEAY"
 url = "{}{}".format(sf.instance_url, path)
 resp = requests.get(url, headers=sf.headers)
 content = json.loads(resp.text)
 print(content)
-the_good_stuff = content["factMap"]["T!T"]["aggregates"][0] # change this to 3 for SMD/FMD
+the_good_stuff = content["factMap"]["T!T"]["aggregates"][3] # change this to 0 for Giving Tuesday
```

Comment:
**suggestion: (non-blocking)** This comment is a funny system of reminding ourselves of how this works 😄. I know it's a pattern you're just building on so nothing wrong with this, but maybe we just keep both comments?

```
# use 0 for Giving Tuesday
# use 3 for SMD/FMD
```

It's a little messy but makes it so we don't have to keep changing it back and forth. 😵

Reply:
Good call... just updated and moved some things around/removed some magic numbers... think it's easier to see what pieces can be updated now

Reply:
Ah yes, much easier to follow! Thank you @matthewdylan